.github/workflows/bcny-firebase.yml

```diff
@@ -287,6 +287,7 @@ jobs:
           path: com.google.firebase.windows.${{ matrix.arch }}.*.nupkg
 
       - name: Publish NuGet Packages
+        if: ${{ github.event_name == 'workflow_dispatch' }}
```

Comment:
Would it make sense to limit the above upload-artifact for `workflow_dispatch` similarly? Would it save storage space?

Reply:
Yeah I think that's a good idea. At first I figured we'd want to keep this upload for debugging but it's easy enough to trigger the workflow manually and inspect the package if needed. Updated in the latest commit.